rest/base_client.go

```diff
@@ -22,6 +22,15 @@ import (
 // was buried in the "internal" package where we couldn't use it.
 var Domain string
 
+// Use a proxy, "" is none. (This is because the flags package doesn't support Optional)
+var UseProxy string
```

Comment:
How about calling this `ProxyName` or `ProxyAddress` instead? If we are only supporting HTTP proxies, then this should also be indicated.

Reply:
Changed in https://github.com/akitasoftware/akita-cli/pull/195/commits/3a73809e34f771b0dc7149c89e0b705710b66d1c